### PR TITLE
31 remove tests

### PR DESCRIPTION
**Pull request by tschm**

### CODE REVIEW

**Comment by tschm** [comment]
jinja2 is needed for the table produced at the end of the experiments

**Comment by tschm** [comment]
I have removed the sys.path hack... :-)

**Comment by tschm** [comment]
I know run 'make experiments during ci/cd'

**Comment by tschm** [comment]
I made sure there is a checkpoints folder :-)

**Comment by tschm** [comment]
And I solved the Secret Santa problem ;-)

**Comment by phschiele** [comment]
@tschm I only added the hack because I wanted the main file that holds the actual reference implementation to be in the root of the repo (markowitz.py). I'm not sure if moving it so a subfolder is preferred here. 

Are you sure we need any of these additional libraries?

**Comment by tschm** [comment]
We need at least jinja2 for now. My experiments crashed when it came to creating the table saying jinja2 is missing. Loguru is not needed yet. Cvxsimulator also not needed at this stage

**Comment by tschm** [comment]
For me experiments.py is the entry point and hence could be top level. I think markowitz.py is better in the experiments "package". Easier and cleaner?

File: experiments/backtest.py [hunk 1]
```diff
@@ -4,16 +4,16 @@
...
 import numpy as np
 import cvxpy as cp
 import pandas as pd
-from utils import synthetic_returns
+from experiments.utils import synthetic_returns
 
+# VERBOTEN
 # hack to allow importing from parent directory without having a package
-sys.path.append(str(Path(__file__).parent.parent))
+# sys.path.append(str(Path(__file__).parent.parent))
```
Comment (phschiele) [inline experiments/backtest.py:16]: Then let's fully remove it instead of commenting it out.

File: requirements.txt [hunk 2]
```diff
@@ -2,3 +2,6 @@ cvxpy
 numpy
 pandas
 matplotlib
+jinja2        # needed to generate the tables for Latex?
+loguru        # professional logging
+cvxsimulator  # for backtesting
```
Comment (phschiele) [inline requirements.txt:7]:
```suggestion
pandas[output-formatting]
matplotlib
```

**Review by phschiele — APPROVED** [review]
Had two small comments, then good to go from my side.

**Comment by tschm** [comment]
committed your changes

**Pull request merged**